doc/Makefile.am

```
@@ -13,4 +13,11 @@ DEFAULT_ERROR_DIR       = $(datadir)/errors
 
 EXTRA_DIST = \
 	debug-messages.dox \
-	debug-sections.txt
+	debug-sections.txt \
+	SPONSORS.in
+
+noinst_data_DATA = SPONSORS
+CLEANFILES = $(data_DATA)
```

The "noinst_data" prefix reads like an oxymoron to me -- "do not install and use datadir when installing".

CLEANFILES appears to use an undefined variable data_DATA. Also, I doubt we should assume that all data_DATA files need to be explicitly mentioned in CLEANFILES.

This needs to be tested, but I hope that we can avoid all three problems using something simple like this:

```suggestion
CLEANFILES = SPONSORS
```

---

This is resolved by [c75fe01](https://github.com/squid-cache/squid/pull/1321/commits/c75fe01389dfec00e5ed06af54fcd01b0fc0bb42)